### PR TITLE
Updating Managing Users, Accounts, and Setup for new login system

### PR DESCRIPTION
**Pull request by brittag**

### CODE REVIEW

File: content/getting-started/accounts.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ You will need a Cloud Foundry account before continuing. To get an account:
 
 ## Resetting your password
 
-Do you have a Cloud Foundry account, but don't know the password? Go to https://login.cloud.gov/forgot_password to reset it. You can also change your password from the command line with `cf passwd`.
+If you log into Cloud Foundry using your agency's account system, you can reset your password using your agency's normal process.
+
+If you log in with a Cloud Foundry account that has its own password (including `ORGNAME_deployer` accounts), you can reset your password by going to https://login.cloud.gov/forgot_password. You can also change your password from the command line with `cf passwd`.
```
Comment (mogul) [inline content/getting-started/accounts.md]: Unfortunately it's actually not possible to set your password from the command-line unless you're an admin! We should be removing the latter sentence. (Ack, I had no idea this was out there; explains a lot!)

File: content/getting-started/setup.md [hunk 2]
```diff
@@ -18,7 +18,15 @@ As a user, nearly all of your interactions with Cloud Foundry will be through th
...
     ```
 
-1. Log in by running
+1. **If you log in using your agency's account system**, log in by running
+
+    ```bash
+    cf login -a api.cloud.gov --sso
+    ```
+    
+    Then follow the passcode link to get your one time code and enter it to log in.
```
Comment (mogul) [inline content/getting-started/setup.md]: Should it be `one-time` with the dash?

Comment (brittag) [inline content/getting-started/setup.md]: Currently the copy outside this documentation is: “One Time Code” in the command line prompt, “Temporary Authentication Code” on the website, and “passcode” in the link URL. It'd be nice to pick the best one and gradually standardize on it. GSA's standard phrasing seems to be "One-Time Registration Code" - should we go with that?

Comment (mogul) [inline content/getting-started/setup.md]: "Registration" is misleading because no account is being registered... How about "One-Time Authentication Code"?

**Comment by brittag** [comment]
In this copy I've taken a couple of steps toward slightly-more-standardized language to reduce confusion: 
- "cloud.gov account" preferred over "Cloud Foundry account"
- "one-time authentication code" preferred over "one time code", "passcode", "temporary authentication code", "one-time registration code", and so on

**Comment by mogul** [comment]
LGTM!

**Pull request merged**